recipe/build.sh

```diff
@@ -1,25 +1,29 @@
 #!/bin/bash
 
-TCLTKVERSION=8.5
+TCLTKVERSION=8.6
 
 # Compile OOMMF.
-export OOMMF_TCL_CONFIG=${PREFIX}/lib/tclConfig.sh
-export OOMMF_TK_CONFIG=${PREFIX}/lib/tkConfig.sh
+export OOMMF_TCL_CONFIG=${SYS_PREFIX}/lib/tclConfig.sh
```

Comment:
This should be able to stay as `${PREFIX}` once the correct `build` and `host` dependencies are specified. `$PREFIX` is the *host* environment prefix, which should contain anything that the package links to, e.g. tk.